client/gatsby-node.js

```diff
@@ -261,7 +261,22 @@ exports.onCreateWebpackConfig = ({ stage, actions }) => {
         process: require.resolve('process/browser')
       }
     },
-    plugins: newPlugins
+    plugins: newPlugins,
+    ignoreWarnings: [
+      warning => {
+        if (warning instanceof Error) {
+          if (
+            warning.message.includes('only default export is available soon')
+          ) {
+            return true;
+          }
+          if (warning.message.includes('mini-css-extract-plugin')) {
+            return true;
+          }
+        }
+        return false;
+      }
+    ]
```

Comment:
I am OK with the mini-css one, but is there a chance that the `only default export is available soon` will suppress something we do not want suppressed?

Reply:
Potentially. If I understand this correctly, Webpack is saying in a future major release, our `import { someEnvVar } from "./path/to/env.json";` will not resolve (import) properly.

To be on the safer side, shall I just keep the `mini-css` one?

Reply:
Yes, lets start there. I know its not pretty but I would rather be safe.